components/apimgt/org.wso2.carbon.apimgt.impl/src/test/resources/dbscripts/h2.sql

```diff
@@ -1496,6 +1496,7 @@ CREATE TABLE IF NOT EXISTS AM_ALERT_TYPES_VALUES (
             ALERT_TYPE_ID INTEGER,
             USER_NAME VARCHAR(256) NOT NULL ,
 	    STAKE_HOLDER VARCHAR(100) NOT NULL ,
+        PRIMARY KEY (ALERT_TYPE_ID),
```

Comment:
These are added in the test db scripts. Please check in the db scripts in **features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql**

Reply:
looks like the same ALERT_TYPE_ID is expected to be in multiple rows. so please check whether this is indeed a primary key.

Reply:
Edited as below : 
AM_ALERT_EMAILLIST_DETAILS - (EMAIL_LIST_ID,EMAIL)
AM_ALERT_TYPES_VALUES - (ALERT_TYPE_ID,USER_NAME,STAKE_HOLDER)